geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/RecordUpdateProcessor.scala

```diff
@@ -165,15 +171,16 @@ trait RecordUpdateProcessor extends DataStoreProcessor {
             }
           })
 
-          session.putAttribute(file, Attributes.IngestSuccessCount, success.toString)
-          session.putAttribute(file, Attributes.IngestFailureCount, failure.toString)
-          session.transfer(file, SuccessRelationship)
+          var output = file
+          output = session.putAttribute(output, Attributes.IngestSuccessCount, success.toString)
+          output = session.putAttribute(output, Attributes.IngestFailureCount, failure.toString)
```

Comment:
Ok, so I haven't read closely enough yet.  What's going on with output here?

Reply:
all the methods on an output flow file return a new (updated) flow file back. i wasn't seeing the attributes come through correctly without referencing the returned flow file in the converter processor, so i updated it here as well. i'm not sure if it was broken before actually? but this seems safer